app/src/layouts/DashboardLayout/Sidebar/index.tsx

```diff
@@ -1,8 +1,11 @@
 import React from "react";
 import { PrimarySidebar } from "./PrimarySidebar";
+import { isAppTypeSessionBear } from "utils/AppUtils";
 import "./Sidebar.css";
 
 export const Sidebar: React.FC = () => {
+  if (isAppTypeSessionBear()) return null;
```

Comment:
handle this [here](https://github.com/requestly/requestly/blob/6db0cbf3eb80ea230a530912e54d514122e44b5d/app/src/layouts/DashboardLayout/index.jsx#L33-L35) instead